src/components/NavBar/NavMenu.jsx

```diff
@@ -52,32 +52,33 @@ const NavMenu = ({ menuId, openMenu, setOpenMenu, anchorEl, setAnchorEl, setShow
             Notifications
           </Button>
         </MenuItem>
-        <MenuItem>
-          <Button
-            variant="text"
-            startIcon={<AccountCircle />}
-            href={session.info.webId}
-            target="_blank"
-            rel="noreferrer"
-          >
-            Profile
-          </Button>
+        <MenuItem
+          component={Button}
+          variant="text"
+          startIcon={<AccountCircle />}
+          href={session.info.webId}
+          target="_blank"
+          rel="noreferrer"
+          sx={{ color: 'rgba(1, 121, 105, 1)' }}
+        >
+          Profile
         </MenuItem>
         <Divider />
-        <MenuItem>
-          <Button variant="text" startIcon={<SettingsIcon />}>
-            Settings
-          </Button>
+        <MenuItem
+          component={Button}
+          variant="text"
+          startIcon={<SettingsIcon />}
+          sx={{ color: 'rgba(1, 121, 105, 1)' }}
+        >
+          Settings
         </MenuItem>
-        <MenuItem>
-          <Button
-            variant="text"
-            color="error"
-            startIcon={<LogoutIcon />}
-            onClick={() => setShowConfirmation(true)}
-          >
-            Logout
-          </Button>
+        <MenuItem
```

Comment:
Think we could also get this `<MenuItem>` to be fullWidth like with "Settings" and "Profile". Makes it look nice and consistent.

Reply:
I believe all three MenuItems are full width. Let me know if I have misunderstood!

Reply:
Similar to what I've seen in the opening post, this is what I saw when testing. 

https://github.com/codeforpdx/PASS/assets/14917816/9db90016-15c1-4fa1-9fca-e86024ceb745

"Setting" and "Profile" looks pretty good having stretched out to the full width of the container, but not for "Log out".

Not really sure what MUI is doing with this component since it does look like it supposed to stretch out, but think we could patch this up by setting `fullWidth` for this `<MenuItem>` like what is done with "Setting".

https://github.com/codeforpdx/PASS/assets/14917816/e144982e-6c7c-415b-be66-955cbf099fbb

Reply:
Aha, I see what you mean now. I was watching the left side of the button and just assumed it was symmetrical. Will take another look at the logout button this evening.